libs/features/mas/web-components/src/variants/ccd-slice.css.js

```diff
@@ -0,0 +1,65 @@
+import { TABLET_UP, DESKTOP_UP, LARGE_DESKTOP } from "../media.js";
+
+export const CSS = `
+:root {
+  --consonant-merch-card-ccd-slice-single-width: 322px;
+  --consonant-merch-card-ccd-slice-icon-size: 30px;
+  --consonant-merch-card-ccd-slice-wide-width: 600px;
+  --consonant-merch-card-ccd-slice-single-height: 154px;
+  --consonant-merch-card-ccd-slice-background-img-size: 119px;
+}
+
+.one-merch-card.ccd-slice,
+.two-merch-cards.ccd-slice,
+.three-merch-cards.ccd-slice,
+.four-merch-cards.ccd-slice {
+    grid-template-columns: var(--consonant-merch-card-ccd-slice-width);
+}
+
+merch-card[variant="ccd-slice"] [slot='body-s'] {
```

Comment:
i wonder if we shouldn't try to move slot styling if possible into variantStyles? cc @yesil

Reply:
I agree, anything that can stay in the shadowRoot should go in variantStyles.